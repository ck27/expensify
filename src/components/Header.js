import React from "react";

import {NavLink} from "react-router-dom";

const Header = () => (
    <header>
        <div>
            <h1>Expensify</h1>
        </div>
        <div className="links">
            <NavLink exact={true} to="/" activeClassName="is-active">DASHBOARD</NavLink>
            <NavLink to="/add" activeClassName="is-active">ADD EXPENSE</NavLink>
            <NavLink to="/edit" activeClassName="is-active">EDIT EXPENSE</NavLink>
            <NavLink to="/help" activeClassName="is-active">HELP</NavLink>
        </div>
    </header>

);

export default Header;