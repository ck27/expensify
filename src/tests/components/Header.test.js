import React from "react";
import {shallow } from 'enzyme'; // ES6
import toJson from "enzyme-to-json";
import Header from "../../components/Header";

test("it must render Header correctly", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
});