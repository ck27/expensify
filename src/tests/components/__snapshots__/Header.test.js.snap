// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it must render Header correctly 1`] = `
<header>
  <div>
    <h1>
      Expensify
    </h1>
  </div>
  <div
    className="links"
  >
    <NavLink
      activeClassName="is-active"
      exact={true}
      to="/"
    >
      DASHBOARD
    </NavLink>
    <NavLink
      activeClassName="is-active"
      to="/add"
    >
      ADD EXPENSE
    </NavLink>
    <NavLink
      activeClassName="is-active"
      to="/edit"
    >
      EDIT EXPENSE
    </NavLink>
    <NavLink
      activeClassName="is-active"
      to="/help"
    >
      HELP
    </NavLink>
  </div>
</header>
`;
